RELEASING.md

```diff
@@ -1,42 +1,39 @@
+
+The `release-prepare` branch will become `main` after the relevant Cosmos Hub Network Upgrade successfully takes place.
+
+## Release Procedure
+
+### Minor & Patch Releases
+
+TODO
+
+### Major Release
+
+For a new major release, `m`, after a previous release, `n`, start on `release/vn.0.x` and checkout a new branch called `upgradename-main`(e.g. `Theta-main`). This will be the `release-prepare` branch referenced above going forward until the network upgrades with the new Major Release. This branch should act like `main` in that it is the target for new PRs that come in with code meant for the upcoming release. Some PRs that target the real `main` will be relevant to this `upgradename-main`, and should similarly be cherry picked to this branch. Before beginning the initial release candidate process, this branch should pass the `dev-testnet` tests. Once passing, checkout a new branch, `release/vm.0.x` off  `release/vn.0.x`. From `release/vm.0.x` checkout two branched: `release/vm.0.0-rc0` and `rc0/vm.0.0-rc0`. Merge `upgradename-main` into `rc0/vm.0.0-rc0`.(not directly merge `upgradename-main` into `release/vm.0.0-rc0` is to protect `upgradename-main` from being deleted, any branch merged to `release/` branch will be deleted after merge). Now open a PR from `rc0/vm.0.0-rc0` against `release/vn.0.0-rc0`. This PR is important as it's the last time code reviewers can see all changes going into the release at once. Last minute changes can go directly into `rc0/vm.0.0-rc0` as well as updating the CHANGELOG.md. Once the PR has been merged, the `rc0/vm.0.0-rc` branch will be automatically deleted. The `release/vm.0.0-rc0` is merged again into the `release/vm.0.x` branch, and tagged, then the release can be published as a pre-release. `release/vm.0.x` will also be automatically deleted.
```

Comment:
I updated this paragraph, due to 1. no more each release is a branch, 2. avoid `Theta-main` being deleted.
issue is as mentioned here 
https://github.com/cosmos/gaia/pull/1327#issuecomment-1070825842

to avoid automatically delete `theta-main`, 
we can either do the above mentioned method, make a temporal branch `rc0/v7.0.0-rc0` off `release/v7.0.0-rc0`
or we can checkout a new branch off `Theta-main`, and merge this new branch to `release/v7.0.0-rc0` branch. and merge `release/v7.0.0-rc0` to `release/v7.0.x`

@okwme please check here for the specific steps presently for v7.0.0-rc0. https://hackmd.io/MEpvsls9T1aanPJWXer6Ig #Release Process.